src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWDG2kFyxJVt_0WjtZ9-&open=AWDG2kFyxJVt_0WjtZ9-)

[ISSUE_KEY]: AWDG2kFyxJVt_0WjtZ9-

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED

Reply:
This issue has been reopened

[ISSUE_STATUS]: REOPENED